app/controllers/api/v1/users_controller.rb

```diff
@@ -15,7 +17,7 @@ def user
       end
 
       def update_params
-        params.require(:user).permit(:name, :gender)
+        params.require(:user).permit(:name, :gender, avatar: %i[data filename content_type])
```

Comment:
Are you expecting a base64 file? If this image is used for an Avatar, why do you need filename and content_type?